### DISCUSSION TITLE
Feature request: Add more commands: !guns, gun, guns

### DISCUSSION THREAD

**Discussion by trollollol123** [yonilerner/cs2-retakes-allocator#22] [Ideas]
Simple, add more commands to make it easier for players. Some plugins use !guns for example, so it muscle memory to type !guns. And then on retakes it doesn't work because its !gun for example.

Also print out availble weapons in chat for players when typing any of the commands above. Good for beginners or people who have a hard time remembering all the weapon names.. The print should show what exactly to type to get the weapon.

Also, maybe make "known" shortcuts for weapons. m4a4 = m4? 

Example: !guns
Available weapons: ak, m4a4, awp (etc..)

**Comment by B3none** [comment]
Adding the `gun` / `guns` commands with no prefix will be fun to do.

**Comment by yonilerner** [comment]
> Also, maybe make "known" shortcuts for weapons. m4a4 = m4?

This is already a thing FYI. As long as any part of it matches it should work. I need to improve the handling for when there are multiple matches

> Also print out availble weapons in chat for players when typing any of the commands above

This is a good idea i should do

**Comment by yonilerner** [comment]
This is all basically done!
